step.yml

```diff
@@ -114,13 +111,28 @@ inputs:
       description: |-
         __Only available if `track` is `rollout`.__
 
-        Portion of the users who should get the staged rollout version of the app.
-      value_options:
-      - "0.05"
-      - "0.1"
-      - "0.2"
-      - "0.5"
+        Portion of the users who should get the staged rollout version of the app. Accepts values between 0.0 and 1.0.
       is_required: true
+      value_options:
+        - "0.05"
+        - "0.1"
+        - "0.15"
+        - "0.2"
+        - "0.25"
+        - "0.3"
+        - "0.35"
+        - "0.4"
+        - "0.45"
+        - "0.5"
+        - "0.55"
+        - "0.6"
+        - "0.65"
+        - "0.7"
+        - "0.75"
+        - "0.8"
+        - "0.85"
+        - "0.9"
+        - "0.95"
```

Comment:
It could be an issue if the user wants to release to all users (so no staging); can they do that at the moment? See https://developers.google.com/android-publisher/tracks.

Reply:
It is not said explicitly on the docs, but I expect to use release status for that (set it to completed) and do not define user fraction. See APK Workflow example here:
https://developers.google.com/android-publisher/tracks

Also docs says this to fraction:
Fraction of users who are eligible to receive the release. 0 < fraction < 1. To be set, release status must be "inProgress" or "halted".
Please see https://developers.google.com/android-publisher/api-ref/edits/tracks

I think fraction should not be a required input then, I update the code accordingly.